jobs/job_runner.go

```diff
@@ -0,0 +1,82 @@
+package jobs
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"api.audius.co/config"
+	"api.audius.co/logging"
+	"go.uber.org/zap"
+)
+
+type JobRunner interface {
+	Execute(ctx context.Context, logger *zap.Logger) error
+}
+
+type BaseJob struct {
```

Comment:
I'm not a fan of this pattern. I think it's fine if each job implements an interface, or shares a common function, but this is more like inheritance.

Reply:
`go new Runner(new MyJob()).ScheduleEvery()` or something